src/test/java/org/prebid/server/auction/ExchangeServiceTest.java

```diff
@@ -1048,10 +1050,44 @@ public void shouldRequestCacheServiceWithExpectedArguments() {
         // then
         verify(cacheService).cacheBidsOpenrtb(
                 argThat(t -> t.containsAll(asList(bid1, bid2))), eq(asList(imp1, imp2)),
-                eq(CacheContext.of(true, null, false, null)),
+                eq(CacheContext.builder().shouldCacheBids(true).videoBidIdsToModify(emptyList()).build()),
                 eq(Account.builder().id("accountId").eventsEnabled(false).build()), eq(timeout));
     }
 
+    @Test
+    public void shouldRequestCacheServiceWithVideoBidsToModifyWhenEventsEnabledAndForBidderThatAllowsModifyVastXml() {
+        // given
+        final Bid bid1 = Bid.builder().id("bidId1").impid("impId1").price(BigDecimal.valueOf(5.67)).build();
+        final Bid bid2 = Bid.builder().id("bidId2").impid("impId2").price(BigDecimal.valueOf(7.19)).build();
+        givenBidder("bidder1", mock(Bidder.class), givenSeatBid(singletonList(givenBid(bid1))));
+        givenBidder("bidder2", mock(Bidder.class), givenSeatBid(singletonList(givenBid(bid2))));
+
+        given(bidderCatalog.isModifyingVastXmlAllowed(eq("bidder1"))).willReturn(true, false);
+
+        // imp ids are not really used for matching, included them here for clarity
```

Comment:
Seems this comment is not relevant here anymore, since we looking though imp.id == bid.impId in `org.prebid.server.auction.ExchangeService#isVideoBid` ?

Reply:
Yes, that's a copy-paste from other case, removed it from couple other places as well where it might be contextually used.